### PR TITLE
Fix for: Repeatable field is no longer rendered with Chosen layout

### PR DESCRIPTION
**Pull request by Fedik**
Pull Request for Issue #16110

### Summary of Changes
The patch rerender Chosen.js if it was there before

### Testing Instructions
please see #16110 and #14556 for detail

Example xml which can be used for test:
```xml
<field name="repeatable" type="repeatable" icon="list" label="Repeatable" select="select">
	<fieldset hidden="true" name="repeatable_modal" repeat="true">

			<field type="list" name="list1" label="List1">
			<option>- Select -</option>
			<option value="val1">val 1</option>
			<option value="val2">val 2</option>
			<option value="val3">val 3</option>
		</field>

		<field type="list" name="list2" label="List2" multiple="true">
			<option>- Select -</option>
			<option value="val4">val 4</option>
			<option value="val5">val 5</option>
			<option value="val6">val 6</option>
			<option value="val7">val 7</option>
			<option value="val8">val 8</option>
		</field>
	</fieldset>
</field>

```

### Expected result
Chosen.js rendered

### Actual result
Only default `<selec>` list 

### Documentation Changes Required
none

### CODE REVIEW

**Comment by Fedik** [comment]
@N6REJ please test also

File: media/system/js/repeatable-uncompressed.js [hunk 1]
```diff
@@ -437,8 +437,10 @@
         	// destroy chosen if any
         	if($.fn.chosen){
                 $row.find('select').each(function(){
-                	if ($(this).data('chosen')) {
-                        $(this).chosen('destroy');
+					var $el = $(this);
+					if ($el.data('chosen')) {
+						$el.chosen('destroy');
+						$el.addClass('here-was-chosen');
```
Comment (laoneo) [inline media/system/js/repeatable-uncompressed.js]: nice class name :smile:

**Comment by roland-d** [comment]
@Fedik I applied the patch but I am still getting a plain select list.

image

**Comment by Fedik** [comment]
@roland-d It will apply Chosen.js only if it available on the page, it do not force one.
I have tested in the Isis template params.

**Comment by roland-d** [comment]
I have tested this item :white_check_mark: successfully on f5337c9a220547f5148acbbb406265fbfd10fef4<br /><br />The dropdown was a regular select before the patch but after applying the patch it is now a chosen dropdown.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16471">issues.joomla.org/tracker/joomla-cms/16471</a>.</sub>

**Comment by N6REJ** [comment]
sorry I missed this, for some reason git is not emailing me with notices.. I'll get this tested in next 72hrs

**Comment by roland-d** [comment]
@N6REJ The 72 hours are up, are you able to test this again?

**Comment by ghost** [comment]
@N6REJ can you please test?

**Comment by N6REJ** [comment]
sorry guys, idk why but github refuses to send me notification for joomla.  I get them for neard but not from joomla.  I keep forgetting to get this done.. I"m going to leave the tab open so I don't forget.. give me 30 min.

**Comment by N6REJ** [comment]
I have tested this item :white_check_mark: successfully on f5337c9a220547f5148acbbb406265fbfd10fef4<br /><br />This worked very nicely.  RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/16471">issues.joomla.org/tracker/joomla-cms/16471</a>.</sub>

**Comment by ghost** [comment]
RTC after two successful tests.

**Pull request merged**